### PR TITLE
fix: modify entities to save evm events

### PR DESCRIPTION
**Pull request by melisaguevara**
This PR:
- Deletes the entities Deposit, Fill and SlowFillRequest that were created before.
- Creates a new schema 'evm' for storing events from evm compatible chains.
- Creates new entities representing some of the Across V3 contract events: V3FundsDeposited, FilledV3Relay and RequestedV3SlowFill.

### CODE REVIEW

File: packages/indexer-database/src/entities/evm/V3FundsDeposited.ts [hunk 1]
```diff
@@ -7,18 +7,12 @@ import {
...
-@Unique("UK_deposit_depositId_originChainId", ["depositId", "originChainId"])
-export class Deposit {
+@Entity({ schema: "evm" })
+@Unique("UK_v3FundsDeposited_depositId_originChainId", [
+  "depositId",
+  "originChainId",
+])
+export class V3FundsDeposited {
   @PrimaryGeneratedColumn()
   id: number;
```
Comment (amateima) [inline packages/indexer-database/src/entities/evm/V3FundsDeposited.ts:18]: Can we be more specific about the uuid column and name it smth like relayHash?

File: packages/indexer-database/src/migrations/1724271636924-V3FundsDeposited.ts [hunk 2]
```diff
@@ -0,0 +1,45 @@
+import { MigrationInterface, QueryRunner } from "typeorm";
```
Comment (amateima) [inline packages/indexer-database/src/migrations/1724271636924-V3FundsDeposited.ts:1]: I think we can remove the previous migration files and any references to the old table names as the app was not deployed yet and we don't have any database to maintain

File: packages/indexer-database/src/entities/evm/V3FundsDeposited.ts (hunk 1 above)
Comment (melisaguevara) [reply packages/indexer-database/src/entities/evm/V3FundsDeposited.ts:17]: Renamed here [a2a2667](https://github.com/across-protocol/indexer/pull/14/commits/a2a2667cb4394afd98f602d53f21e30e1d284a16)

File: packages/indexer-database/src/migrations/1724271636924-V3FundsDeposited.ts (hunk 2 above)
Comment (melisaguevara) [reply packages/indexer-database/src/migrations/1724271636924-V3FundsDeposited.ts]: Deleted here [a2a2667](https://github.com/across-protocol/indexer/pull/14/commits/a2a2667cb4394afd98f602d53f21e30e1d284a16)

**Review by daywiss — APPROVED** [review]
lgtm

**Pull request merged**